Makefile

```diff
@@ -11,26 +11,49 @@ help:
 	@echo "db: initializes the sqlite database"
 	@echo ""
 
-deps:
-	@echo "Installing dependencies"
-	@echo "-----------------------"
+.PHONY: aptdeps
+aptdeps:
+	@echo "Installing apt dependencies"
+	@echo "---------------------------"
 	sudo apt-get update
-	sudo apt-get install docker-engine npm nodejs-legacy python3-pip python3.4-venv
+	sudo apt-get install -o Dpkg::Options::="--force-confold" --force-yes -y docker-engine npm nodejs-legacy python3-pip python3.4-venv
+	sudo pip install --upgrade pip
 	sudo npm install -g configurable-http-proxy
-	python3 -mvenv venv
-	. venv/bin/activate && pip3 install docker-py
+
+.PHONY: pythondeps
+pythondeps:
+	@echo "Installing dependencies"
+	@echo "-----------------------"
+	if test -z "${NO_VENV}"; \
```

Comment:
If you just mean to see if the make file is being used on travis, you can check the `TRAVIS` environment variable. https://docs.travis-ci.com/user/environment-variables/#Default-Environment-Variables